docs/reference/configuration/RTOS.md

```diff
@@ -19,7 +21,11 @@ Name: rtos.present
     Defined by: library:rtos
     Macro name: MBED_CONF_RTOS_PRESENT
     Value: 1 (set by library:rtos)
+```
 
+### Event configuration parameters
```

Comment:
That's "Event queue" https://os.mbed.com/docs/v5.9/reference/eventqueue.html

Reply:
It's not "event": https://os.mbed.com/docs/v5.9/reference/event.html? That's confusing

Reply:
Event class represents and event in the event queue. The config options refer to the settings of EventQueue.